change/@uifabric-experiments-2019-07-15-17-47-12-chicletPreviewImageFixes.json

```
@@ -0,0 +1,8 @@
+{
+  "comment": "Chiclet: removes imageWidth and imageHeight props",
+  "type": "patch",
```

I believe in this case it’s a `minor` change because it’s modifying the API by removing props. Theoretically is a `major` breaking change, but due to it being an experimental playground package I think we can stick with minor.
```suggestion
  "type": "minor”,
```